homeassistant/components/inels/translations/cs.json

```diff
@@ -0,0 +1,48 @@
+{
```

Comment:
Please remove foreign languages from the PR and only keep English. Other languages can be managed from our translation platform, once this PR has been merged.